thermal_base/flyr_unpack.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3

```

Comment:
You can remove this shebang. Makes it incompatible in OSes/environments/Windows/etc